README.md

```diff
@@ -58,23 +58,7 @@ and push it
 ➜ cf push
 ```
 
-#Development
-The project uses the standard spring boot directory structure, with web content located
```

Comment:
Ahh – I think I missed commit in my clean-up.   It needs to auto-run the web pack task.

Tom

From: Joshua Kruck <notifications@github.com<mailto:notifications@github.com>>
Reply-To: krujos/willitconnect <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Monday, November 9, 2015 at 8:55 AM
To: krujos/willitconnect <willitconnect@noreply.github.com<mailto:willitconnect@noreply.github.com>>
Cc: Thomas Gamble <Thomas_Gamble@homedepot.com<mailto:Thomas_Gamble@homedepot.com>>
Subject: Re: [willitconnect] Merged the other two pull requests and cleaned up some additional js (#11)


In README.md<https://github.com/krujos/willitconnect/pull/11#discussion_r44276373>:

> @@ -58,23 +58,7 @@ and push it
>  ➜ cf push
>  ```
>
> -#Development
> -The project uses the standard spring boot directory structure, with web content located


did we change how bootRun works?

—
Reply to this email directly or view it on GitHub<https://github.com/krujos/willitconnect/pull/11/files#r44276373>.

________________________________

The information in this Internet Email is confidential and may be legally privileged. It is intended solely for the addressee. Access to this Email by anyone else is unauthorized. If you are not the intended recipient, any disclosure, copying, distribution or any action taken or omitted to be taken in reliance on it, is prohibited and may be unlawful. When addressed to our clients any opinions or advice contained in this Email are subject to the terms and conditions expressed in any applicable governing The Home Depot terms of business or client engagement letter. The Home Depot disclaims all responsibility and liability for the accuracy and content of this attachment and for any damages or losses arising from any inaccuracies, errors, viruses, e.g., worms, trojan horses, etc., or other items of a destructive nature, which may be contained in this attachment and shall not be liable for direct, indirect, consequential or special damages in connection with this e-mail message or its attachment.